requirements-dev.txt

```diff
@@ -1,5 +1,7 @@
 depthai-sdk==1.13.1
-viam-sdk==0.11.0
+viam-sdk==0.25.2
+Pillow==10.4.0
```

Comment:
Pillow was removed as an implicit dep in our Python SDK https://github.com/viamrobotics/viam-python-sdk/pull/567